pro/cypress/e2e/features/createAccount.feature

```diff
@@ -0,0 +1,8 @@
+@P0
+Feature: Account creation
+
+  Scenario: It should create an account
+    Given I open "inscription" page
+    When I fill required information in create account form
```

Comment:
Est-ce que pour les formulaires ça n'est pas problématique d'encapsuler tout dans une grosse étape ?
Par ex si je suis le métier je n'ai aucune idée de comment les champs sont remplis sans aller lire le code des steps

Est-ce qu'on metterait pas des steps paramétriques du genre `I type "LEMOINE" in the "Nom" text field`

Reply:
Comme précisé dans la description: `la syntaxe choisie est la version "compacte", sans expliciter les valeurs utilisées dans la couche gherkin, à voir si on favorisera les data table par la suite`

Donc on utilisera surement les data table ce qui fait que les paramètres seront bien visibles dans le feature